integration-tests/jiva/invalidconfig/invalidconfig_test.go

```diff
@@ -0,0 +1,194 @@
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package invalidconfig
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+
+	ns "github.com/openebs/maya/pkg/kubernetes/namespace/v1alpha1"
+	pvc "github.com/openebs/maya/pkg/kubernetes/persistentvolumeclaim/v1alpha1"
+	sc "github.com/openebs/maya/pkg/kubernetes/storageclass/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	// defaultReplicaLabel represents the jiva replica
+	defaultReplicaLabel = "openebs.io/replica=jiva-replica"
+	// defaultCtrlLabel represents the jiva controller
+	defaultCtrlLabel                  = "openebs.io/controller=jiva-controller"
+	openebsProvisioner                = "openebs.io/provisioner-iscsi"
+	accessModes                       = []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce}
+	capacity                          = "5G"
+	scObj                             *storagev1.StorageClass
+	pvcObj                            *corev1.PersistentVolumeClaim
+	nsObj                             *corev1.Namespace
+	openebsCASConfigValue             = "- name: ReplicaCount:\n  Value: 3"
+	pvcLabel, replicaLabel, ctrlLabel string
+)
+
+var _ = Describe("TEST INVALID CAS CONFIGURATIONS IN SC", func() {
+	var (
+		namespaceName = "validation-ns1"
+		scName        = "jiva-invalid-config-sc"
+		pvcName       = "jiva-volume-claim"
+	)
+	BeforeEach(func() {
+		annotations := map[string]string{
+			string(apis.CASTypeKey):   string(apis.JivaVolume),
+			string(apis.CASConfigKey): openebsCASConfigValue,
+		}
+		var err error
+
+		nsObj, err = ns.NewBuilder().
+			WithName(namespaceName).
+			APIObject()
+		Expect(err).ShouldNot(HaveOccurred())
+
+		scObj, err = sc.NewBuilder().
+			WithName(scName).
+			WithAnnotations(annotations).
+			WithProvisioner(openebsProvisioner).Build()
+		Expect(err).ShouldNot(HaveOccurred())
+
+		pvcObj, err = pvc.NewBuilder().
+			WithName(pvcName).
+			WithNamespace(namespaceName).
+			WithStorageClass(scName).
+			WithAccessModes(accessModes).
+			WithCapacity(capacity).Build()
+		Expect(err).ShouldNot(HaveOccurred())
+
+		By(fmt.Sprintf("Creating test specific namespace {%s}", namespaceName))
+		_, err = ops.nsClient.Create(nsObj)
+		Expect(err).To(BeNil())
+
+		By(fmt.Sprintf("Creating PVC named {%s} in Namespace: {%s}", pvcName, namespaceName))
+		_, err = ops.pvcClient.WithNamespace(namespaceName).Create(pvcObj)
+		Expect(err).To(BeNil())
+
+		pvcLabel = string(apis.PersistentVolumeClaimKey) + "=" + pvcName
```

Comment:
Can these be set at the same location where names are set

Reply:
Done